docs/dev/docker-hub.md

```diff
@@ -25,21 +25,6 @@ Production grade configuration requires tuning a few kernel parameters
 such that limiting the number of available cores (with `--smp 1`) is
 the simplest way to go.
 
-Using multiple cores requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
-On many non-production systems, it will be equal to 65K. The formula
-to calculate the proper value is:
-
-    Available AIO on the system - (request AIO per-cpu * ncpus) =
-    aio_max_nr - aio_nr < (reactor::max_aio + detect_aio_poll + reactor_backend_aio::max_polls) * cpu_cores =
-    aio_max_nr - aio_nr < (1024 + 2 + 10000) * cpu_cores =
-    aio_max_nr - aio_nr < 11026 * cpu_cores
-
-    where
-
-    reactor::max_aio = max_aio_per_queue * max_queues,
-    max_aio_per_queue = 128,
-    max_queues = 8.
-
```

Comment:
Is this formula documented anywhere else? Is it still valid?
I removed it as I think it is it unclear and doesn't add value to the user. 
Perhaps it should be here or in a different place in Docs but written more clearly with explanations.

Reply:
not documented as "document" but as source code, i think Ivan added it by inspecting the related source code, see https://github.com/scylladb/seastar/blob/ae94791773a57d37023774b2d88f4a2a5c84ed64/src/core/reactor.cc#L4169-L4180 in Seastar back in f4412029f4144a7c63784b8cd042683cfdf35370.

so it is valid. but it is confusing. yeah, we do need address the problem originally supposed to be addressed by f4412029f4144a7c63784b8cd042683cfdf35370. we can offer an formula like:

```python
(max_networking_io_control_blocks + max_storage_io_control_blocks + max_preempt_io_control_blocks) * num_cpu_cores
```

where

* `max_networking_io_control_blocks` is used for performing networking I/O, its value is 10000 by default and can be configured using option with the same name,
* `max_storage_io_control_blocks` is used for performing storage I/O. its value is hardwired to 1024
* `max_preempt_io_control_blocks` is used by the Seastar framework internally for preempting tasks. its value is 2.
* num_cpu_cores is the number of processing units assigned to a Seastar application, in this case, Scylla. it can be configured using the `smp` option.

please note, this formula and the explanations applies to all Seastar applications. 

@dgarcia360 i plan to add [role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html) support to the configuration options, so we can reference them using hyper link, what do you think?